aws/data_source_aws_eks_cluster.go

```diff
@@ -70,6 +70,14 @@ func dataSourceAwsEksCluster() *schema.Resource {
 							Computed: true,
 							Elem:     &schema.Schema{Type: schema.TypeString},
 						},
+						"endpoint_private_access": {
```

Comment:
Please add acceptance testing for these two attributes, e.g.

```go
resource.TestCheckResourceAttrPair(resourceName, "vpc_config.0.endpoint_private_access", dataSourceResourceName, "vpc_config.0.endpoint_private_access"),
resource.TestCheckResourceAttrPair(resourceName, "vpc_config.0.endpoint_public_access", dataSourceResourceName, "vpc_config.0.endpoint_public_access"),
```

Nit: Attribute alphabetical ordering